### ISSUE TITLE
Order Confirmation email sent while redirecting to payment gateway 

### ISSUE THREAD

**Issue by shubhamaxtrics** [payu-india/PayUbiz_Magento#11]
An order confirmation email is being sent while redirecting to the payment gateway.
Scenario:
Go to checkout -> fill details -> click on online payments -> redirecting to payu payment gateway 
At this point, it sends an email Order confirmation before adding my card details.
Magento 2.3.2

**Comment by shahnawaz123** [comment]
Yes, I am facing same issue order confirmation email trigger before placing the order.
 
2.3.5 M2
